src/components/web3-connect/index.tsx

```diff
@@ -9,20 +9,21 @@ import { ConnectionStatus, Connectors } from '../../lib/web3';
 import { setAddress, setConnectionStatus, updateConnector } from '../../store/web3';
 
 export interface Properties {
-  connectionStatus: ConnectionStatus;
-  setConnectionStatus: (status: ConnectionStatus) => void;
-  setAddress: (address: string) => void;
-  updateConnector: (connector: Connectors) => void;
-  providerService: { register: (provider: any) => void };
-  connectors: { get: (connector: Connectors) => any };
-  currentConnector: Connectors;
+  connectionStatus: ConnectionStatus,
```

Comment:
we have added prettier for this purpose, @hkhanouchi  can you please run `npm run code-format:fix` and check if it will change comma to semi-colons ?

if it's solving the problem -> maybe we should run `npm i` on every pull

Reply:
ok.